File: dotnet-backend/Models/Operatore.cs
namespace dotnet_backend.Models;

public class Operatore
{
    public int? id { get; set; }
    public string? ruolo { get; set; }
    public string? nome { get; set; }
    public string? cognome { get; set; }
    public string? username { get; set; }
    public string? password { get; set; }
    public int sede_id { get; set; }

}

File: node_backend/model/models/Operatore.js
class Operatore {
  id;
  nome;
  cognome;
  ruolo;  
  username;
  password;
  sede_id;
   constructor(rawUtente) {
    this.id = rawUtente.id;
    this.nome = rawUtente.nome;
    this.cognome = rawUtente.cognome;
    this.ruolo = rawUtente.ruolo;
    this.username = rawUtente.username;
    this.password = rawUtente.password;
    this.sede_id = rawUtente.sede_id;
  }
  getPublicFields() {
    return {
      id: this.id,
      nome: this.nome,
      cognome: this.cognome,     
      ruolo: this.ruolo,
      username: this.username,
      sede_id: this.sede_id
    }
  }
}

module.exports = Operatore;